source/configuration/write.txt

```diff
@@ -140,6 +140,12 @@ The following options for writing to MongoDB are available:
          ``false``.
        |
        | **Default:** ``true``
+   
+   * - ``convertJson``
+     - | When ``true``, the Connector parses the string and converts extended JSON 
```

Comment:
[suggestion] I think this should be lower case to match the way "connector" is used on the rest of the page. It is only capitalized when we say "MongoDB Spark Connector".